app/controllers/browse_controller.rb

```diff
@@ -31,6 +35,14 @@ def shelf_list_params
       params.permit(:starting, :ending, :length, :nearby)
     end
 
+    def lc_shelf_list_params
+      shelf_list_params.merge(classification: 'lc')
+    end
+
+    def dewey_shelf_list_params
+      shelf_list_params.merge(classification: 'dewey')
+    end
+
```

Comment:
add `:classification` to the list of permitted params and you can use the original method.

Reply:
done